tools/trap/ramdumpParser.py

```diff
@@ -727,14 +727,31 @@ def find_crash_point(log_file, elf):
 
     # It displays the debug symbols corresponding to all the addresses in the kernel and application text address range
     print('\nStack_address\t Symbol_address\t Symbol location  Symbol_name\t\tFile_name')
-    os.system("python ../debug/debugsymbolviewer.py " + log_file + " " + str(g_app_idx) + " 0")
+    os.system("python3 ../debug/debugsymbolviewer.py " + log_file + " " + str(g_app_idx) + " 0")
 
     print('\n4. Miscellaneous information:')
     # It displays the debug symbols corresponding to all the wrong sp addresses (if any)
     # The last argument to debugsymbolviewer specifies whether or not to check for wrong stack pointer addresses
-    os.system("python ../debug/debugsymbolviewer.py " + log_file + " " + str(g_app_idx) + " 1")
+    os.system("python3 ../debug/debugsymbolviewer.py " + log_file + " " + str(g_app_idx) + " 1")
     print('-----------------------------------------------------------------------------------------')
 
+# Function to delete the timestamp from the log file if it consists of timestamp at the start of each line
+def format_log_file(log_file):
+
+	with open(log_file, "r") as f:
+		data = f.readlines()
+	with open(log_file, "w") as f:
```

Comment:
can we open file in read/write mode once

Reply:
Modified, thanks

Reply:
I'm opening it in once in read mode & then in write in order to replace the read contents into the file. 
With read/write(r+) mode, will need to use file operations.